readthedocs/core/tests/test_signals.py

```diff
@@ -0,0 +1,56 @@
+import pytest
+
+from django.contrib.auth.models import User
+from django_dynamic_fixture import G
+
+from readthedocs.oauth.models import RemoteOrganization
+from readthedocs.projects.models import Project
+
+
+@pytest.mark.django_db
+class TestProjectOrganizationSignal(object):
```

Comment:
I'm mostly certain we have tests for both of these cases already. Can you provide test output of these tests without your patch applied?

Reply:
Maybe is worth it to add a test for the `delete_account` view too

Reply:
Yeah, I am going to add some test on that! but its out of scope of this PR!
will open another PR for that!

Reply:
Great, test output looks good. That's what I'd expect if the delete cascaded to the project